src/labelle/lib/devices/device_config.py

```diff
@@ -50,19 +55,57 @@ def matches_device_id(self, idValue: int) -> bool:
         else:
             return False
 
-    def get_tape_print_margins(self, tapeSizeMM: int) -> tuple[int, int]:
+    def get_tape_print_size_and_margins_px(
+        self, tapeSizeMM: int
+    ) -> tuple[int, int, int]:
         """Get print margins for a specific tape size.
 
         :param tapeSizeMM: Tape size in mm
-        :return: Margins tuple, None if not supported
+        :return: Margins tuple in pixels [ActivePixels,TopMarginPx,BottomMarginPx]
         """
         if tapeSizeMM in self.supportedTapeSizes:
-            if tapeSizeMM in self.marginsPerTape:
-                # Return specified margins
-                return self.marginsPerTape[tapeSizeMM]
+            # Calculate the pixels per mm for this printer
+            # Example: printhead of 128 Pixels, distributed over 18 mm of active area.
+            #   Makes 7.11 pixels/mm
+            printPPmm: float = (
+                self.printHeadSizePixels / self.printHeadActiveAreaMillimeters
+            )
+
+            # Calculate usable tape width (*2 for top and bottom)
+            usableTapeWidth: float = tapeSizeMM - (
+                self.TAPE_ALIGNMENT_INACCURACY_MM * 2
```

Comment:
I guess this is a really silly comment, but since in my mind I read "tape width minus twice the margins", I would write `2 * self.TAPE_ALIGNMENT_INACCURACY_MM` instead of `self.TAPE_ALIGNMENT_INACCURACY_MM * 2`

Reply:
If It makes you happy, I'll change it

Reply:
Changed